### PR TITLE
Remove TheVoid

### LINKED ISSUE #228: Solve the mysteries of TheVoid
**Describe the bug**

Clean up this mess.

https://github.com/jellyfin/jellyfin-kodi/blob/2f9fb660cb9a894517131eb56312c2488e9d0fd9/jellyfin_kodi/downloader.py#L349-#L386

https://github.com/jellyfin/jellyfin-kodi/blob/2f9fb660cb9a894517131eb56312c2488e9d0fd9/jellyfin_kodi/entrypoint/default.py#L275-#L314

**Additional context**

Cleaning this madness will likely solve #205 and #217

### PR DESCRIPTION
**Pull request by mcarlton00**
Removes TheVoid.  Builds off the first half of #416 and closes #228.

The short explanation is that it now initializes the apiclient in `entrypoint/default.py` and then passes that object around to any functions that need it.  I _think_ it's all working.  You should test.  Seriously. Test things.  Or yolo merge it, whatever.  I'm not your mom.

### CODE REVIEW

**Comment by oddstr13** [comment]
I am getting this stack trace when attempting to play items in the dynamic menu – I don't really use those parts regularly, so I don't know if that's due to this pr.
```python
2020-11-20 15:03:04.915 T:31164    INFO <general>: JELLYFIN.helper.playutils -> INFO::jellyfin_kodi/helper/playutils.py:92 Skip source selection.
2020-11-20 15:03:04.917 T:31164    INFO <general>: JELLYFIN.helper.playutils -> INFO::jellyfin_kodi/helper/playutils.py:197 --[ direct stream ]
2020-11-20 15:03:07.874 T:31164    INFO <general>: JELLYFIN.objects.actions -> INFO::jellyfin_kodi/objects/actions.py:132 [ intro/aebdcd16ced445cda245c2f4033b4de0 ] Water Splash
2020-11-20 15:03:07.876 T:31164    INFO <general>: JELLYFIN.__main__ -> ERROR::default.py:37 'NoneType' object has no attribute 'config'
                                                   Traceback (most recent call last):
                                                     File "default.py", line 35, in <module>
                                                       Events()
                                                     File "jellyfin_kodi/entrypoint/default.py", line 91, in __init__
                                                       Actions(server, api_client).play(item, params.get('dbid'), params.get('transcode') == 'true', playlist=params.get('playlist') == 'true')
                                                     File "jellyfin_kodi/objects/actions.py", line 57, in play
                                                       self.set_playlist(item, listitem, db_id, transcode)
                                                     File "jellyfin_kodi/objects/actions.py", line 102, in set_playlist
                                                       self._set_intros(item)
                                                     File "jellyfin_kodi/objects/actions.py", line 134, in _set_intros
                                                       play = playutils.PlayUtils(intro, False, self.server_id, self.server)
                                                     File "jellyfin_kodi/helper/playutils.py", line 67, in __init__
                                                       'Token': api_client.config.data['auth.token']
                                                   AttributeError: 'NoneType' object has no attribute 'config'
```

**Review by oddstr13 — APPROVED** [review]
Begone!

**Comment by mcarlton00** [comment]
It was broken because of this PR.  Failing when trying to get intros from the server.  Last commit _should_ have fixed it, but I don't have any to test with and confirm

File: jellyfin_kodi/jellyfin/__init__.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def __init__(self, server_id=None):
         self.server_id = server_id or "default"
 
     def get_client(self):
+        # type: () -> JellyfinClient
```
Comment (oddstr13) [inline jellyfin_kodi/jellyfin/__init__.py:57]: :heart_eyes:

**Review by oddstr13 — APPROVED** [review]
I have tested the following (rebased on master):
- :heavy_check_mark: Full clean sync (`rm -r ~/.kodi/{media,system,userdata,temp}`.
- :heavy_check_mark: Content added while Kodi is running.
- :heavy_check_mark: Content added while Kodi is not running.
- :heavy_check_mark: Playback of Movies and TV Episodes.
- :heavy_check_mark: Playback via the dynamic menus.

I have not tested music sync due to #428.

**Pull request merged**